### DISCUSSION TITLE
pfSense+tpws без эффекта на SmartTV

### DISCUSSION THREAD

**Discussion by max-arturo** [bol-van/zapret#298] [Q&A] [answered]
Доброго дня всем.
К pfSense подкрутил tpws, завернув весь трафик на него (пока без hostlist'ов).
На всех устройствах в локальной сети ютуб стал работать как в лучшие годы, но на телевизорах приложение по-прежнему не открывается.

Может быть у кого есть идеи о причинах такого поведения?

**Comment by bol-van** [comment, marked as answer]
вероятная причина в использовании теликом tls 1.2
он может не обходиться tpws
блокчеком можно проверить  tpws на рутракере и посмотреть что может сделать  tpws для tls 1.2 и 1.3
обычно в лучшем случае на 1.2 срабатывает mss, но это нехорошая стратегия
с замедлением у них тот же детектор, только включается не блок, а тротлинг

**Reply by bol-van** [reply]
решение в использвании фейков, но это умеет только dvtws

**Reply by max-arturo** [reply]
> вероятная причина в использовании теликом tls 1.2 он может не обходиться tpws блокчеком можно проверить tpws на рутракере и посмотреть что может сделать tpws для tls 1.2 и 1.3 обычно в лучшем случае на 1.2 срабатывает mss, но это нехорошая стратегия с замедлением у них тот же детектор, только включается не блок, а тротлинг

Прогнал блокчек для рутрекера, результат такой:
```
ipv4 rutracker.org curl_test_https_tls12 : tpws not working
ipv4 rutracker.org curl_test_https_tls12 : dvtws --dpi-desync=fake,split2 --dpi-desync-ttl=1
```
Буду пробовать с dvtws

**Reply by bol-van** [reply]
о чем и речь

**Reply by ub9ybi** [reply]
получилось?

**Reply by max-arturo** [reply]
> получилось?

да, через dvtws всё получилось, поэтому от tpws пока отказался

**Reply by ub9ybi** [reply]
на ТВ идет а компы облом значит пока.

**Reply by ub9ybi** [reply]
такое не пробовали https://dtf.ru/u/1811008-andrey/2910376-uskoryaem-youtube-bez-ustanovki-softa ? или это тоже самое?

**Reply by max-arturo** [reply]
> такое не пробовали https://dtf.ru/u/1811008-andrey/2910376-uskoryaem-youtube-bez-ustanovki-softa ? или это тоже самое?

то же самое
